src/vs/workbench/contrib/notebook/common/services/notebookSimpleWorker.ts

```diff
@@ -56,7 +56,7 @@ class MirrorCell {
 		hashValue = doHash(this.language, hashValue);
 		hashValue = doHash(this.getValue(), hashValue);
 		hashValue = doHash(this.metadata, hashValue);
-		hashValue = doHash(this.internalMetadata, hashValue);
+		hashValue = doHash({ ...this.internalMetadata, 'cellId': '' }, hashValue);
```

Comment:
Temporary work around for issue caused by https://github.com/microsoft/vscode/pull/240733
Either we revert the PR or just add a work around and fix the known issues, 
Going with the latter